src/cryptography/x509.py

```diff
@@ -5,13 +5,14 @@
 from __future__ import absolute_import, division, print_function
 
 import abc
+import datetime
 import ipaddress
 from enum import Enum
 
 import six
 
 from cryptography import utils
-from cryptography.hazmat.primitives import hashes
+from cryptography.hazmat.primitives import dsa, ec, rsa, hashes
```

Comment:
`dsa`, `ec`, and `rsa` all come from `cryptography.hazmat.primitives.asymmetric`